portfolio/src/main/webapp/script.js

```diff
@@ -194,6 +191,12 @@ function prevVideo() {
     }
     // Decrement the analysis index and update analysis page elements.
     const curAnalysis = playlistAnalyses[--analysisIndex];
+    updateAnalysis(curAnalysis);
+}
+
+// Update the analysis container with a new video analysis.
+function updateAnalysis(curAnalysis) {
+    document.getElementById("analysis-title").textContent = curAnalysis.videoTitle;
     document.getElementById("happy-meter").value = curAnalysis.sentimentScore;
     google.search.cse.element.getElement("analysis-search").execute(curAnalysis.searchQueryString);
```

Comment:
Could we get some constants for these ID Names? Otherwise LGTM!

Reply:
Fixed!